app/components/auk/abbreviated-text.hbs

```diff
@@ -0,0 +1,23 @@
+{{#if this.textTooLong}}
+  <div class="auk-o-flex">
+    {{#if (has-block)}}
+      {{yield this.abbreviatedText}}
+    {{else}}
+      {{this.abbreviatedText}}
+    {{/if}}
```

Comment:
die has-block, is die nodig als je `<Auk::AbbreviatedText .......  as |abbreviatedText|>` gaat doen? Of wat doet die juist

Reply:
Het zoek-vak rechts bovenaan de Ember API-guides is uw vriend :wink: ([has-block](https://api.emberjs.com/ember/3.26/classes/Ember.Templates.helpers/methods/hasBlock?anchor=hasBlock))

Reply:
Heb het eens bekeken en ben wel mee nu :sweat_smile: